packages/xrpl/src/models/ledger/Ledger.ts

```diff
@@ -63,3 +58,40 @@ export default interface Ledger {
    */
   transactions?: Array<Transaction & { metaData?: TransactionMetadata }>
 }
+
+/**
+ * A ledger is a block of transactions and shared state data. It has a unique
+ * header that describes its contents using cryptographic hashes.
+ *
+ * @category Ledger Entries
+ */
+export interface Ledger extends BaseLedger {
+  /**
+   * The ledger index of the ledger. Represented as a native JSON number.
```

Comment:
nit: weirdly worded to say "native json number"

Reply:
Yeah, I can replace it with just "number".